py_gtfs_rt_ingestion/lib/postgres_utils.py

```diff
@@ -86,35 +141,54 @@ def get_local_engine(echo: bool = False) -> sa.engine.Engine:
         raise exception
 
 
-def insert_metadata(written_file) -> None:  # type: ignore
+def _rds_writer_process(metadata_queue: Queue) -> None:
     """
-    add a row to metadata table containing the filepath and its status as
-    unprocessed
+    process for writing matadata paths recieved from metadata_queue
+
+    if None recieved from queue, process will exit
```

Comment:
rds writer process, just waits for items to come in queue and writes them to metadata table